### PR TITLE
Adds missing firelocks to Z1

### PR DESCRIPTION
**Pull request by Mewchild**
Adds firelocks around the more recent mapping additions

### CODE REVIEW

**Comment by Mewchild** [comment]
I'll probably go around later and check every door on station, but this will do for now. A fire escaping into the tram stops it from launching.

File: vorestation.dme [hunk 1]
```diff
@@ -2618,17 +2629,6 @@
 #include "code\modules\rogueminer_vr\wrappers.dm"
 #include "code\modules\rogueminer_vr\zone_console.dm"
 #include "code\modules\rogueminer_vr\zonemaster.dm"
-#include "code\modules\research\designs\ai_modules.dm"
-#include "code\modules\research\designs\circuit_assembly.dm"
-#include "code\modules\research\designs\circuits.dm"
-#include "code\modules\research\designs\illegal.dm"
```
Comment (Arokha) [inline vorestation.dme]: Ohm, what is this

Comment (Mewchild) [reply vorestation.dme]: Oh, I forgot to unstage this. It's something that popped up like... a week ago, when I did a fetch/merge.

**Comment by Mewchild** [comment]
Something went fucky along the line. I don't know. Too tired, will simply redo

**Pull request closed without merging**